models/db/user.ts

```diff
@@ -6,11 +6,13 @@ interface User {
   calc_records: number;
   email: string;
   isOfficial: boolean;
+  last_visited_at?: Date; // last time user visited website
```

Comment:
should this be a number instead of Date? can also call it last_visited_ts so we know it's a timestamp